### PR TITLE
[6] Product Page using HTML

### PR DESCRIPTION
**Pull request by KutrCB**
## Task: `[6] Product Page using HTML`
- `added an index file and images to create my first product page`
Screenshot from 2020-07-21 21-04-01
Screenshot from 2020-07-21 21-03-12

### CODE REVIEW

File: submissions/charlesBanda/html/index.html [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+		</header>
+		<div>
+			<table>
+				<tr>
+					<th>
+		          <h1><font face="Arial">Msika</font></h1>
+								<p><em>Home/Clothing/Stylish men and women winter gloves</p>
+		           </th>
+		           <!--form search bar-->
+		              <form action="process.php" method="post">
```
Comment (ToluAkin) [inline submissions/charlesBanda/html/index.html:27]: Remove attributes `action` and `post` from the form tag.

File: submissions/charlesBanda/html/index.html [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+		           	<input type="text" name="">
+		           </form>
+		        </tr>
+		    </table>
+	    </div>
+			<hr>
+			<!--product image details-->
+			<h3>Stylish Men and Women Winter Gloves</h3>
+    <figure>
+		  <img src="img/img_glove1.jpg" ALT="Black thick warm winter gloves for Men and women"
```
Comment (ToluAkin) [inline submissions/charlesBanda/html/index.html:38]: Remove  the styling of `height` and `width` from the images.

File: submissions/charlesBanda/html/index.html [hunk 3]
```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
```
Comment (ToluAkin) [inline submissions/charlesBanda/html/index.html:1]: Well done!

**Comment by KutrCB** [comment]
Alright working on it

**Pull request closed without merging**